bin/yaml/swift.yaml

```diff
@@ -36,6 +36,17 @@ compilers:
           - '5.9'
           - '5.10'
           - '6.0.3'
+    dev-snapshot:
+      dir: swift-dev-snapshot
+      type: restQueryTarballs
+      install_always: true
```

Comment:
question for the maintainers: Swift typically has ~2 or so active pre-release branches per year. these generally are built with about the same frequency as the nightlies, but once a new release branch is cut, updated builds for old ones will cease (presumably).

so i'm wondering – is `install_always` appropriate here? i didn't entirely chase down the logic that it entails... but i guess the concern would be if the development snapshot version here becomes out of date (since it will require manual updating by someone currently), it might end up wasting bandwidth.

Reply:
Hi @jamieQ - thanks for the question. `install_always` does indeed force it to install every time without checking.

For other nightlies we check a dated directory and so only install at most once a day, but that's special cased for the installer types that are "nightly" in nature. I think we can start with `install_always` and then work from there though if it becomes a problem.